spec/operators/scan-spec.ts

```diff
@@ -231,14 +231,14 @@ describe('Observable.prototype.scan', () => {
   it('should accept array typed reducers', () => {
```

Comment:
Even test name seems we just copied test but haven't updated? :scream:

Reply:
The test name still seemed to make sense to me. I'm not entirely certain the tests are actually fully correct either. I'll leave references in the actual code.

Reply:
yeah what I'm saying is these test seems like just copied from reduce and we haven't updated at all. Not saying test case name should be updated as well.

Reply:
Ah k. :D